src/main/scala/firrtl/passes/AnnotateValidMemConfigs.scala

```diff
@@ -206,24 +202,23 @@ case class SRAMCompiler(
     bestRule.getValidConfig(width = memWidth, depth = memDepth).get.copy(xsplit = validConfig.xsplit, ysplit = validConfig.ysplit)
   }
 
-  def serialize() = {
+  def serialize = {
     // TODO
   }
```

Comment:
if this is TODO in the sense that it shouldn't be called, you can use ??? and it'll error if someone calls it

ie.
```scala
def serialize = ???
```